tests/unit/mocha/filter-test-modules-test.js

```diff
@@ -0,0 +1,126 @@
+import { convertFilePathToModulePath, filterTestModules } from 'ember-exam/test-support/-private/filter-test-modules';
+import { describe, it, beforeEach, afterEach } from 'mocha';
+import { expect } from 'chai';
+
+describe('Unit | filter-test-modules', () => {
+  describe('convertFilePathToModulePath', () => {
+    it('should return an input string without file extension when the input contains file extension', () => {
+      expect(convertFilePathToModulePath('/tests/integration/foo.js')).to.deep.equal(
```

Comment:
Since this is string comparison (and not deep object comparison), we can use `to.equal` instead.
see https://www.chaijs.com/api/bdd/#method_equal